src/index.js

```diff
@@ -0,0 +1,99 @@
+import './style.css';
+// import Icon from './icon.png';
+/*
+function component() {
+    const element = document.createElement('div');
+
+    // Lodash, now imported by this script
+    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+    element.classList.add('hello');
+
+    // Add the image to our existing div.
+    const myIcon = new Image();
+    //myIcon.src = Icon;
+
+    element.appendChild(myIcon);
+
+    //return element;
+  }
+
+  document.body.appendChild(component()); */
+
+/*
+  <label class="todo-list-label">
+  <input type="checkbox">
+  <p>NEW ITEM</p>
+  <input type="button" value="X">
+</label> */
```

Comment:
## OPTIONAL
- Please remove these commented lines of codes to keep your code clean and looks more professional.